source/features/hide-inactive-deployments.tsx

```diff
@@ -2,11 +2,14 @@ import select from 'select-dom';
 import features from '../libs/features';
 
 function init(): void {
-	const deployments = select.all('.TimelineItem .deployment-meta');
+	const deployments = select.all(`
+		.TimelineItem [title^="Deployment Status Label:"],
+		.js-socket-channel[data-url$="/pull_requests/events/deployed"] .TimelineItem a.btn
+	`);
```

Comment:
```suggestion
	const deployments = select.all(`.TimelineItem [title^="Deployment Status Label: Inactive"]`);
```

Reply:
I meant like this. Skip the condition altogether, just select what you need to delete.